src/styles/nav.css

```diff
@@ -0,0 +1,127 @@
+.label {
+  display: none;
+}
+.label-hide {
+  display: inline;
+}
+#toogle {
+  display: none;
+}
+.navbars {
+  display: flex;
+  height: 60px;
+  justify-content: space-between;
+  width: 100%;
+  background: linear-gradient(145deg, #f5f5f5, #504a9e);
+  box-shadow: 5px 5px 7px #c5c5c5, -5px -5px 7px #fbfbfb;
+}
+.navbars > ul {
+  display: flex;
+  margin-right: 5rem;
+}
+.navbars > ul > div {
+  text-align: center;
+  padding: 0.75rem;
+}
+.side-bar {
+  text-align: left;
+}
+.side-bar > ul {
+  display: flex;
+  justify-content: flex-start;
+}
+.side-bar > ul > li {

```

Comment:
As earlier avoid complex selectors . Its also difficult to maintain. Use class names instead.